waiter/src/waiter/handler.clj

```diff
@@ -536,6 +538,16 @@
     (catch Exception ex
       (utils/exception->response ex request))))
 
+(defn get-last-request-times-state
+  "Outputs the last-request-time state."
+  [router-id last-request-time-agent request]
+  (try
+    (-> {:router-id router-id
+         :state @last-request-time-agent}
+        (utils/map->streaming-json-response))
+    (catch Exception ex
```

Comment:
Now that we have error handling middleware, we don't necessarily need to handle the exception here, especially given that we aren't adding decorating the exception with `:status`.